pages/common/wget.md

```diff
@@ -4,11 +4,11 @@
 > Supports HTTP, HTTPS, and FTP.
 > More information: <https://www.gnu.org/software/wget>.
 
-- Download the contents of an URL to a file (named "foo" in this case):
+- Download the contents of a URL to a file (named "foo" in this case):
 
 `wget {{https://example.com/foo}}`
 
-- Download the contents of an URL to a file (named "bar" in this case):
+- Download the contents of a URL to a file (named "bar" in this case):
```

Comment:
I dont think that's right

Reply:
It's different depending on how you pronounce `URL` (`you are el` gets `a`, not sure about the rest), but most of the time (and this time), `a URL` is correct.